core/domain/services/graphql/types/Datetime.php

```diff
@@ -306,5 +307,35 @@ public function registerMutations(array $inputFields)
                 'mutateAndGetPayload' => DatetimeCreate::mutateAndGetPayload($this->model, $this),
             ]
         );
+        
+        // Register mutation to update an entity.
+        register_graphql_mutation(
+            'reordoer' . $this->namespace . 'Entities',
+            [
+                'inputFields'         => [
+                    'entityIds'  => [
+                        'type'        => [
+                            'non_null' => ['list_of' => 'ID'],
+                        ],
+                        'description' => esc_html__('List of entity guids in the order that is desired.', 'event_espresso'),
+                    ],
+                    'entityType' => [
+                        'type'        => [
+                            'non_null' => $this->namespace . 'ModelNameEnum',
+                        ],
+                        'description' => esc_html__('The entity type of the IDs', 'event_espresso'),
+                    ],
+                ],
+                'outputFields'        => [
+                    'ok' => [
+                        'type'    => 'Boolean',
+                        'resolve' => function ($payload) {
+                            return (bool) $payload['ok'];
+                        },
+                    ],
+                ],
+                'mutateAndGetPayload' => EntityReorder::mutateAndGetPayload(),
+            ]
+        );
```

Comment:
@tn3rb I have registered the mutation in Datetime type class. May be you can find a better place for that.